boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -239,6 +251,43 @@ def trim_index(index, kind, item):
     # Get rid of these so we can iterate over what's left at the end
     trim_index(by_kind, kind, 'ALL')
 
+
+csv['spec']['install']['spec']['permissions'] = []
+
+if 'namespace' not in csv['metadata']:
+    raise UndefinedCSVNamespace(OPERATOR_NAME)
+
+# Find namespace of operator from CSV template
+namespace = csv['metadata']['namespace']
+
+if 'Role' in by_kind:
+    filtered_roles = []
+    filtered_rolebindings = []
+    for role in by_kind['Role']:
+        # We assume there is always a rolebinding of defined role
+        if role['metadata']['name'] not in rb_by_role:
+            raise NoAssociatedRoleBinding(role['metadata']['name'], role['metadata']['namespace'])
+        role_binding = rb_by_role[role['metadata']['name']]
+
+        # If the RoleBinding subject is a ServiceAccount in the same namespace as the operator
+        # we add it to CSV. Else it is to be written as-is to the bundle
+        if len(role_binding['subjects']) == 1 and \
+            role_binding['subjects'][0]['kind'] == 'ServiceAccount' and \
```

Comment:
It would be neat if [discover_service_account](https://github.com/openshift/boilerplate/pull/141/files#diff-e12cb709d3181a1ca667b183451c41bbb410dd055344a17d1c0bc981430fde16R145) returned the whole sa so you wouldn't have to make assumptions like "exactly one subject" etc. But that's a nontrivial change as we would have to dummy one up if not found. Anyway, something to consider for later.